releasenotes/notes/add-diffrax-solvers-946869d5a304318a.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Added support for solvers from the diffrax package:
+    https://github.com/patrick-kidger/diffrax. A new option 
+    is enabled to pass in an object -- a solver from diffrax
+    instead of a string for a jax or scipy solver, for example::
+    
+      from diffrax import Dopri5
+      from qiskit-dynamics.solvers.solver_functions import solve_ode
+
+      sol = solve_ode(
```

Comment:
Is the `example::` valid sphinx?

For this it looks like a code example but you have used a type-hinting syntax, which seems odd. Would it be better to just do:
```
sol = solve_ode(
    rhs,
    t_span,
    y0,
    method=Dopri5()
)
```

Reply:
Are you saying to remove the imports as well?

Reply:
should I remove the import lines as well?

Reply:
Maybe keep the imports as it shows where things are coming from, but you need to fix the `solve_ode` one:
```
from qiskit_dynamics import solve_ode
```
This part is pretty minor as all release notes from other changes get merged into a single file for a release, and at this stage they typically get re-edited anyway.